supplementary/devices/index.rst

```diff
@@ -120,7 +121,7 @@ Devices
      - Navigation
      - Sensors
      - Other
-   * - :doc:`Samsung Galaxy S8+ <dream2lte>`
```

Comment:
why? arent you removing the links all over the document?

Reply:
when i forked a ton of the links changed to just markup; i don't know why, tried to make them human readable